features/networking/multus.feature

```diff
@@ -416,4 +416,118 @@ Feature: Multus-CNI related scenarios
+    And evaluation of `pod` is stored in the :pod clipboard
+    
+    #Clean-up required to erase bridge interfcaes created on node after this step
+    Given I register clean-up steps:
+    """
+    the bridge interface named "mybridge" is deleted from the "<%= cb.pod.node_name %>" node
+    the bridge interface named "mybridge.200" is deleted from the "<%= cb.pod.node_name %>" node
+    """
+    When I execute on the pod:
+      | /usr/sbin/ip | -d | link |
+    Then the output should contain: 
+      | net1 |
+    #Entering into corresponding node to make sure VLAN ID information shown for interfaces
+    Given CNI vlan info is obtained on the "<%= cb.pod.node_name %>" node
+    Then the step should succeed
+    And the output should contain:
+      | 200 |
+    
+  # @author anusaxen@redhat.com
+  # @case_id OCP-24467
+  @admin
+  Scenario: CNO manager mavlan configured manually with static
+    # Make sure that the multus is Running
+    Given the multus is enabled on the cluster
+    Given the default interface on nodes is stored in the :default_interface clipboard 
+    #Patching simplemacvlan config in network operator config CRD
+    Given as admin I successfully merge patch resource "networks.operator.openshift.io/cluster" with:
+      | {"spec":{"additionalNetworks":[{"name":"test-macvlan-case3","namespace":"openshift-multus","simpleMacvlanConfig":{"ipamConfig":{"staticIPAMConfig":{"addresses": [{"address":"10.128.2.100/23","gateway":"10.128.2.1"}]},"type":"static"},"master":"<%= cb.default_interface %>","mode":"bridge"},"type":"SimpleMacvlan"}]}} |
+    #Cleanup for bringing CRD to original
+    Given I register clean-up steps:
+    """
+    Given as admin I successfully merge patch resource "networks.operator.openshift.io/cluster" with: 
+    | {"spec":{"additionalNetworks": null}} |
+    """
+ 
+    And I wait up to 30 seconds for the steps to pass:
+    """
+    When I run the :get admin command with:
+      | resource | net-attach-def   |
+      | n        | openshift-multus |
+    Then the step should succeed
+    And the output should contain:
+      | test-macvlan-case3 |
+    """
+    #Creating pod under openshift-multus project to absorb above net-attach-def
+    Given I switch to cluster admin pseudo user
+    Given I use the "openshift-multus" project
+    When I run oc create over "https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/networking/multus-cni/Pods/1interface-macvlan-bridge.yaml" replacing paths:
+      | ["metadata"]["annotations"]["k8s.v1.cni.cncf.io/networks"] | test-macvlan-case3 |
+    Then the step should succeed
+    And a pod becomes ready with labels:
+      | name=macvlan-bridge-pod |
+    And evaluation of `pod` is stored in the :pod clipboard
+
+    When I execute on the pod:
+      | /usr/sbin/ip | -d | link |
+    Then the output should contain "net1"
+    And admin ensures "<%= cb.pod.name %>" pod is deleted from the "openshift-multus" project after scenario
```

Comment:
this needs to be registered in scenario before pod or immediately after pod is created. In case you need to obtain pod name after creation, then after creation you could use step `a pod is present with labels:`. Then register clean-up and then wait for it to become ready and whatever other tests.

The reason is to make sure failures do not lead to leaked resources.